tests/tests_functionals/test_snr.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
```

Comment:
You can ditch this first line in all files you've committed. It's not required anymore in modern pythons